### ISSUE TITLE
RUNPOD Frame processor frame_enhancer could not be loaded

### ISSUE THREAD

**Issue by 1Tomber** [ashleykleynhans/facefusion-docker#5]
```shell
[FACEFUSION.PROCESSORS.FRAME.CORE] Frame processor frame_enhancer could not be loaded
``` 
can be found when firing up the runpod template in the facefusion.log and facefusion doesn't start.
Tested with and without Network Storage.

**Comment by ashleykleynhans** [comment]
Apologies, fixed in 2.3.2 release.

**Comment by 1Tomber** [comment]
No apologies please, thank you for sharing your work!

Just tested, working great. Thanks for the quick fix!
